tests/contrib/aiohttp/test_templates.py

```diff
@@ -1,19 +1,21 @@
 import asyncio
 import aiohttp_jinja2
 
-from aiohttp.test_utils import unittest_run_loop
-
 from ddtrace.pin import Pin
 from ddtrace.contrib.aiohttp.patch import patch, unpatch
 
 from .utils import TraceTestCase
 from .app.web import set_filesystem_loader, set_package_loader
+from ..asyncio.utils import mark_asyncio_no_close as mark_asyncio
 
 
 class TestTraceTemplate(TraceTestCase):
     """
     Ensures that the aiohttp_jinja2 library is properly traced.
     """
+    def setUp(self):
+        super().setUp()
+        asyncio.set_event_loop(self.loop)
```

Comment:
I think this can be removed as I updated `TraceTestCase`.  Also from test_request_safety